docs/test_indexing.md

```diff
@@ -0,0 +1,10 @@
+1. Create a publishing file in Alma (Either in the sandbox or production)
+    1. Create a Query that contains the data you would like to query.  Save the query after you have it correct
+    2. Change the DRDS test publishing job (Under resoruces) to point at your saved query
+    3. Run the job by clicking run under the ... menu after the save
+1. Download the file from the Alma SFTP.  If you ran the job from the sanbox look in the sanbox directory otherwise the file should be created at the top directory.
+1. Unzip and rename the file to something that makes sense to you locally `<file name>.xml`
+1. sftp the up to the staging worker machine
```

Comment:
Deploy your changes to bibdata-staging
`scp <local file> to deploy@bibdata-alma-worker-staging1:home/deploy:`
`ssh deploy@bibdata-alma-worker-staging1`
`cd /opt/marc_liberation/current`